src/PlatformSockets.c

```diff
@@ -308,6 +307,11 @@ int resolveHostName(const char* host, int family, int tcpTestPort, struct sockad
     for (currentAddr = res; currentAddr != NULL; currentAddr = currentAddr->ai_next) {
         // Use the test port to ensure this address is working
         if (tcpTestPort != 0) {
+#ifdef __vita__
+            if (currentAddr->ai_family == AF_INET6) {
```

Comment:
Does newlib respect the `AI_ADDRCONFIG` flag? If so, we shouldn't get any IPv6 addresses here.

Reply:
technically, vita only can handle ipv4. but actually newlib doesn't handle `AI_ADDRCONFIG` flag.
the problem of `resolveHostName` is we sending the value for `family`.
it means sometimes we can send `AF_INET6` for this.
(maybe we should do early exiting if platform is vita;
```
int resolveHostName(...) {
#ifdef __vita__
  if (family != AF_INET) {
    return -1;
  }
#endif
  struct addrinfo ...
  ...
}
```
)

Reply:
I think it should work without any Vita-specific checks.

Nothing currently passes `AF_INET6`, but if something did, `getaddrinfo()` would just fail with `EAI_FAMILY` and the code would continue as if IPv6 isn't available (which is true on Vita). So I think we should remove the Vita-specific check here.

Reply:
hm. you are right. I'll remove it tonight.